src/test/kotlin/com/hibob/project/dao/FeedbackDaoTest.kt

```diff
@@ -0,0 +1,73 @@
+import java.time.LocalDateTime
+
+@BobDbTest
+class FeedbackDaoTest @Autowired constructor(private val sql: DSLContext) {
+
+    private val feedbackDao = FeedbackDao(sql)
+    private val feedbackTable = FeedbackConfigurationTable.instance
+    private val companyIdTest = 1234L
+    private val userTest = Employee(1, "Adi", "Finkelman", Role.EMPLOYEE, companyIdTest, "development")
+
+    @Test
+    fun `submit feedback successfully`() {
+        val feedbackCreation = FeedbackCreationRequest("Complaint", Timestamp.valueOf(LocalDateTime.now().withNano(0)), false, StatusType.UNREVIEWED)
+        val feedbackId = feedbackDao.feedbackSubmission(feedbackCreation, userTest)
+        val feedback = extractToFeedbackConfiguration(feedbackCreation, feedbackId, userTest.id, userTest.companyId)
+        assertEquals(listOf(feedback), feedbackDao.getAllFeedbacks(companyIdTest))
+    }
+
+    @Test
+    fun `submit multiple feedbacks and get all feedbacks successfully`() {
+        val feedbackCreation1 = FeedbackCreationRequest("Complaint", Timestamp.valueOf(LocalDateTime.now().withNano(0)), false, StatusType.UNREVIEWED)
+        val feedbackCreation2 = FeedbackCreationRequest("Regards", Timestamp.valueOf(LocalDateTime.now().withNano(0)), false, StatusType.UNREVIEWED)
+        val feedBackId1 = feedbackDao.feedbackSubmission(feedbackCreation1, userTest)
+        val feedBackId2 = feedbackDao.feedbackSubmission(feedbackCreation2, userTest)
+        val feedback1 = extractToFeedbackConfiguration(feedbackCreation1, feedBackId1, userTest.id, userTest.companyId)
+        val feedback2 = extractToFeedbackConfiguration(feedbackCreation2, feedBackId2, userTest.id, userTest.companyId)
+        assertEquals(listOf(feedback1, feedback2), feedbackDao.getAllFeedbacks(companyIdTest))
+    }
+
+    @Test
+    fun `submit anonymous feedback successfully`() {
+        val feedbackCreation = FeedbackCreationRequest("Anonymous", Timestamp.valueOf(LocalDateTime.now()), true, StatusType.UNREVIEWED)
+        feedbackDao.feedbackSubmission(feedbackCreation, userTest)
+        assertTrue(feedbackDao.getAllFeedbacks(companyIdTest)[0].isAnonymous)
+    }
+
+    @Test
+    fun `get all feedbacks where list is empty`() {
+        val feedbacks = emptyList<FeedbackConfiguration>()
+        assertEquals(feedbacks, feedbackDao.getAllFeedbacks(companyIdTest))
+    }
+
+    private fun extractToFeedbackConfiguration(feedbackCreationRequest: FeedbackCreationRequest, id: Long, employeeId: Long, companyId: Long): FeedbackConfiguration {
+        val feedbackConfiguration = FeedbackConfiguration(
+            id = id,
+            employeeId = employeeId,
+            companyId = companyId,
+            text = feedbackCreationRequest.text,
+            creationTime = feedbackCreationRequest.creationTime,
+            isAnonymous = feedbackCreationRequest.isAnonymous,
+            status = feedbackCreationRequest.status
+        )
+        return feedbackConfiguration
+    }
+
+    @BeforeEach
```

Comment:
don't use both BeforeEach and AfterEach on the same method. It runs it twice.

(I know you were taught to do that, but it makes it run twice)